### PR TITLE
RYA-127 Mongo support in merge tool.

### PR DESCRIPTION
**Pull request by isper3at**
## Description

> What Changed?
> Added mongo implementation for the merge tool
### Tests

> Coverage?
> Coverage only for the adapter, the store will be tested in integration
### Links

[Jira](https://issues.apache.org/jira/browse/RYA-127)
### Checklist
- [ ] Code Review
- [x] Squash Commits
#### People To Reivew

@ejwhite992

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -49,7 +48,6 @@ out/
 *.iws
 *.ipr
 .clover
-hs_err_pid*.log
```
Comment (ejwhite922) [inline .gitignore:52]: Leave this and the one above it.

File: extras/rya.export/export.mongo/src/main/java/org/apache/rya/export/mongo/MongoRyaStatementStore.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    private final DB db;
+
+    private final String ryaInstanceName;
+    private final MongoClient client;
+    private final MongoDBRyaDAO dao;
+
+    /**
+     * Creates a new {@link MongoRyaStatementStore}.
+     * @param client - The client to connect to Mongo.
+     * @param ryaInstance - The rya instance to connect to.
```
Comment (ejwhite922) [inline extras/rya.export/export.mongo/src/main/java/org/apache/rya/export/mongo/MongoRyaStatementStore.java]: Add @param dao

File: extras/rya.export/export.mongo/src/main/java/org/apache/rya/export/mongo/time/TimeMongoRyaStatementStore.java [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+public class TimeMongoRyaStatementStore extends MongoRyaStatementStoreDecorator {
+    private final Date time;
+    private final DB db;
+
+    private final SimpleMongoDBStorageStrategy adapter;
+
+    /**
+     * @param client
+     * @param ryaInstance
+     * @param time
```
Comment (ejwhite922) [inline extras/rya.export/export.mongo/src/main/java/org/apache/rya/export/mongo/time/TimeMongoRyaStatementStore.java]: Fix params

File: .gitignore (hunk 1 above)
Comment (isper3at) [reply .gitignore:52]: done

File: extras/rya.export/export.mongo/src/main/java/org/apache/rya/export/mongo/MongoRyaStatementStore.java (hunk 2 above)
Comment (isper3at) [reply extras/rya.export/export.mongo/src/main/java/org/apache/rya/export/mongo/MongoRyaStatementStore.java]: done

**Comment by ejwhite922** [comment]
Looks good :ship:

File: extras/rya.export/export.mongo/src/main/java/org/apache/rya/export/mongo/time/TimeMongoRyaStatementStore.java (hunk 3 above)
Comment (isper3at) [reply extras/rya.export/export.mongo/src/main/java/org/apache/rya/export/mongo/time/TimeMongoRyaStatementStore.java]: done

**Pull request merged**